scripts/get_chrome_url

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+
+import json
+import requests
+
+OS = 'linux'
+CHANNEL = 'beta'
+
+BUILDS_URL = 'https://storage.googleapis.com/chromium-browser-continuous/Linux_x64/{}/chrome-linux.zip'
+VERSIONS_URL = 'https://omahaproxy.appspot.com/all.json'
+
+
+def get_build_no(data):
+    for platform in data:
+        if platform['os'] == OS:
+            for version in platform['versions']:
+                if version['channel'] == CHANNEL:
+                    return version['branch_base_position']
```

Comment:
Ugh, this nesting is kind of hard to read. :smile: Should we perhaps flatten it by converting it to a _collection pipeline_ (as per Martin Fowler in his [artcile](http://martinfowler.com/articles/refactoring-pipelines.html))?


```py
from functools import partial
from operator import itemgetter
...

# construct mapping and filtering helper functions
valid_platforms = partial(filter, lambda platform: platform['os'] == OS)
valid_versions = partial(filter, lambda ver: ver['channel'] == CHANNEL)
platf_versions = partial(map, itemgetter('versions'))
...

def get_build_no(data):
    versions = valid_versions(platf_versions(valid_platforms(data)))  # <-- the pipeline itself
    return next(versions, {}).get('branch_base_position')
```